omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -324,7 +324,24 @@
                            isValid = false;
                         }  
               });
-
+                 <!--Not to book on a public holiday-->
+               var publicHolidayURL = window.location.origin + '/' + OPENMRS_CONTEXT_PATH + '/ws/rest/v1/ugandaemr/publicHoliday/?date=' + formattedDate ;
```

Comment:
@reagan-meant The url ws/rest/v1/ugandaemr/publicHoliday/?date=31/07/2020 gives a 404 error

Reply:
Done...Rectified and tested and it is now working....It was an oversight of the forward slash after publicholiday